packages/core/examples/webLoader/hardcodedMetaDataProvider.ts

```diff
@@ -0,0 +1,54 @@
+// Add hardcoded meta data provider for color images
+export default function hardcodedMetaDataProvider(type, imageId, imageIds) {
+  const colonIndex = imageId.indexOf(':');
+  const scheme = imageId.substring(0, colonIndex);
+  if (scheme !== 'web') return;
+
+  if (type === 'imagePixelModule') {
+    const imagePixelModule = {
+      pixelRepresentation: 0,
+      bitsAllocated: 24,
+      bitsStored: 24,
+      highBit: 24,
+      photometricInterpretation: 'RGB',
+      samplesPerPixel: 3,
+    };
+
+    return imagePixelModule;
+  } else if (type === 'generalSeriesModule') {
+    const generalSeriesModule = {
```

Comment:
General Series Module has Series Instance UID, Series Number and Series Description, all of which are used in various places - even if you just add dummy values for them it would be good enough.  Series Data/Time might also be needed for sorting in display sets.